examples/send_file.rs

```diff
@@ -61,6 +61,7 @@ fn internal_server_error() -> Response<Body> {
         .unwrap()
 }
 
+#[allow(unreachable_code)] // TODO: remove this when https://github.com/rust-lang/rust/issues/64636 fixed.
```

Comment:
Also, as https://github.com/rust-lang/rust/issues/64636 has been fixed, we can remove this.